src/shipit_pulse_listener/shipit_pulse_listener/listener.py

```diff
@@ -67,14 +72,47 @@ def parse_payload(self, payload):
         data = payload.get('data')
         assert isinstance(data, dict)
         assert 'heads' in data
-
+        assert 'pushlog_pushes' in data
         logger.info('Received new pushes', revisions=data['heads'])
+
+        # Check pushlog has a list of supported files
+        supported_files = [
+            len(self.list_pushlog_files(pushlog['push_full_json_url'])) > 0
+            for pushlog in data['pushlog_pushes']
+        ]
+        if supported_files and not any(supported_files):
+            logger.warn('No supported file detected, skipping task')
```

Comment:
Warnings and Error levels are reported to sentry. Do you want to receive this log in sentry? 
if Yes, then you probably want some more data about the task
If No, then maybe you can make use info level.